static/app/components/assigneeSelector.tsx

```diff
@@ -296,17 +296,41 @@ class AssigneeSelector extends React.Component<Props, State> {
   renderNewDropdownItems(): ItemsBeforeFilter {
     const teams = this.renderNewTeamNodes();
     const members = this.renderNewMemberNodes();
+    const suggestedAssignees = this.renderSuggestedAssigneeNodes() ?? [];
+    const assigneeIds = new Set();
```

Comment:
instead of constructing the set by iterating thru the suggestedAssignees by filtering, we can do something like 
```suggestion
    const assigneeIds = new Set(suggestedAssignees.map(assignee => `${assignee.value.type}:${assignee.value.assignee.id}`));
```

this way you don't have to do the filter on lines 306-308